python/generate-pypi-package.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Needs to be done beforehand:
+# docker run -e PLAT=manylinux2014_x86_64 -it quay.io/pypa/manylinux2014_x86_64
+#cd /tmp
+#git clone https://github.com/CeetronSolutions/opm-common.git
+#cd opm-common
+#git checkout system-python-pypi
```

Comment:
I am very skeptical to this; the build system should be self contained - with no need for magical setups in advance.

Reply:
I'm really not sure how I ended up making a pull request for all this and I am closing it with my apologies. It isn't ready yet for merging.

It is still very much in a prototype stage.

Reply:
No worries